src/lib.rs

```diff
@@ -156,3 +197,39 @@ pub fn datetime(string: &str) -> Result<DateTime, String> {
         Err(format!("Parser Error: {}", string))
     }
 }
+
+/// Parses a duration string.
+///
+/// A string starts with `P` and can have one of the following formats:
+///
+/// * `P1Y2M3DT4H5M6S`
+/// * `P1W`
+/// * `P<datetime>`
```

Comment:
With above comment this might then turn into something like:

```suggestion
/// * Fully-specified duration without designators characters: `P<datetime>`
```

Not happy with the wording here though.

Reply:
Changed to `Fully-specified duration in DateTime format:` with a link to the struct, is this sufficient?